client/my-sites/drafts/draft-list.jsx

```diff
@@ -152,4 +182,11 @@ var Drafts = React.createClass( {
 	}
 } );
 
-module.exports = DraftList;
+module.exports = connect( ( state, { siteID } ) => {
```

Comment:
While we're here making changes, should we change this to `export default` ?

I also dislike the naming of this prop enough that I'd be fine with destructuring renaming `{ siteID: siteId }` 😆  Or changing the way the prop is passed from ancestor components. Small thing though.